app.js

```diff
@@ -1,8 +1,14 @@
 const app = require("express")()
-const { getTopics, getArticles } = require('./controllers.js')
+const { getTopics, getArticles, getArticlesById } = require('./controllers.js')
 
 app.get('/api/topics', getTopics)
 app.get('/api/articles', getArticles)
+app.get('/api/articles/:article_id', getArticlesById)
+
+app.use(({ status, message }, request, response, next) => {
```

Comment:
Rather than destructuring here, I would leave your error parameter whole, which allows you to pass it on through the error handling middleware chain in the event that it doesn't have a status and message, using `next(err)`